test/integration/consul-container/libs/cluster/network.go

```diff
@@ -20,6 +20,7 @@ func createNetwork(t TestingT, name string) (testcontainers.Network, error) {
 			Name:           name,
 			Attachable:     true,
 			CheckDuplicate: true,
+			SkipReaper:     isRYUKDisabled(),
```

Comment:
This fixes a problem I've been seeing locally where RYUK would still run even when setting the env var to turn it off.